client/src/shared-modules/Header.js

```diff
@@ -1,11 +1,37 @@
 import React from "react";
+import { Link } from "react-router-dom";
 
 function Header() {
-  return (
-    <div>
-      <h1>Header place holder</h1>
-    </div>
-  );
+	return (
+		<div >
+			<header>
+				<a href="#">Collective Foundation</a>
+
+				<nav>
+					<ul>
+						<li>
+							<Link to="/page1">Page 1</Link>
+						</li>
+						<li>
+							<Link to="/page2">Page 2</Link>
+						</li>
+						<li>
+							<Link to="/admin">Admin</Link>
```

Comment:
We do not want the users to view Admin page. Admin page should not be in the menu.